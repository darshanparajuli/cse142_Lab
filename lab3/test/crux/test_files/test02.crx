// test redefinition

var a: int;
var a:float;

func main() : void {
  ::println();
}
