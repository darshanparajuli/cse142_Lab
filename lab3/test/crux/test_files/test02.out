Error parsing file.
DeclareSymbolError(4,5)[a already exists.]
Symbol(readInt)
Symbol(readFloat)
Symbol(printBool)
Symbol(printInt)
Symbol(printFloat)
Symbol(println)
Symbol(a)


